mythril/laser/smt/bitvec.py

```diff
@@ -61,7 +61,7 @@ def __add__(self, other) -> "BitVec":
         union = self.annotations + other.annotations
         return BitVec(self.raw + other.raw, annotations=union)
 
-    def __sub__(self, other) -> "BitVec":
+    def __sub__(self, other: Union[int, "BitVec"]) -> "BitVec":
```

Comment:
`other` can't be `int`. If it is int somewhere, in that case we need to correct it :sweat_smile: